### ISSUE TITLE
1500 error

### ISSUE THREAD

**Issue by stagerap** [aoberoi/cordova-plugin-opentokjs#24]
When viewing a existing stream It only streams audio and i get 1500 error for the video, than it eventually exits completely. When trying to publish a stream i also get the 1500 error and the camera never display's even though it asks me for permission. I have iosrtc and this plugin installed. Is there anything i am missing?

**Comment by iisdan** [comment]
Did you solve this?

**Comment by stagerap** [comment]
Yeah, It was a long time ago and can't remember exactly what I did but pretty sure after adding ssl encryption (https) it solved it. I remember switching off this plugin though because the video quality wasn't good. I've been working on my desktop version lately but plan to complete the cordova app with a separate opentok plugin. Haven't tested the quality yet but this fork on the original songz plugin is updated to nearly the latest opentok library: https://github.com/Taracque/cordova-plugin-opentok
